testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -142,12 +142,12 @@ def test_rmsd_matrix_with_superimposition(self, ens1):
         conf_dist_matrix = encore.confdistmatrix.conformational_distance_matrix(
             ens1,
             encore.confdistmatrix.set_rmsd_matrix_elements,
-            select="name CA",
+            select="name CA and resnum 1:3",
```

Comment:
The final output is insensitive to just using resnum 1:3 as opposed to using the full protein??!?!? What are we even testing here? Are the encore tests so insensitive/random?

Reply:
These tests here are not comparing to a reference array but rmsd calculated with `rms.RMSD` (see l150).

For the tests that have a set of reference results, it's indeed different if I use resnum 1:3